src/js/components/Meter/Circle.js

```diff
@@ -13,12 +13,17 @@ const Circle = forwardRef((props, ref) => {
   const theme = useContext(ThemeContext);
   const width =
     size === 'full' ? 288 : parseMetricToNum(theme.global.size[size] || size);
-  const height =
-    type === 'pie'
-      ? width / 2
-      : parseMetricToNum(theme.global.edgeSize[thickness] || thickness);
+  let height;
+  if (type === 'pie') {
+    height = width / 2;
+  } else if (type === 'semicircle') {
+    height =
+      2 * parseMetricToNum(theme.global.edgeSize[thickness] || thickness);
+  } else
+    height = parseMetricToNum(theme.global.edgeSize[thickness] || thickness);
   const mid = width / 2;
-  const radius = width / 2 - height / 2;
+  const radius =
```

Comment:
I would think we want the "radius" to be the same, since the arc should have the same radius.